packages/server/lib/cloud/studio/StudioLifecycleManager.ts

```diff
@@ -275,6 +275,11 @@ export class StudioLifecycleManager {
         cloudDataSource,
         cfg,
         debugData,
+      }).then((studioManager) => {
+        // eslint-disable-next-line no-console
+        console.log('Studio manager reloaded')
+
+        return studioManager
       }).catch((error) => {
         debug('Error during reload of studio manager: %o', error)
```

Comment:
I wonder if this should be a `console.error`.

Reply:
Updated: [`b940c75` (#31824)](https://github.com/cypress-io/cypress/pull/31824/commits/b940c75909a128c1d2e3be109ff3b5793589df13)